test/parallel/test-util-internal.js

```diff
@@ -7,30 +7,24 @@ const fixtures = require('../common/fixtures');
 const { internalBinding } = require('internal/test/binding');
 
 const {
-  getHiddenValue,
-  setHiddenValue,
-  arrow_message_private_symbol: kArrowMessagePrivateSymbolIndex
+  privateSymbols: {
+    arrow_message_private_symbol,
+  },
 } = internalBinding('util');
 
-assert.strictEqual(
-  getHiddenValue({}, kArrowMessagePrivateSymbolIndex),
-  undefined);
-
 const obj = {};
-assert.strictEqual(
-  setHiddenValue(obj, kArrowMessagePrivateSymbolIndex, 'bar'),
-  true);
-assert.strictEqual(
-  getHiddenValue(obj, kArrowMessagePrivateSymbolIndex),
-  'bar');
+assert.strictEqual(obj[arrow_message_private_symbol], undefined);
+assert.deepStrictEqual(Reflect.ownKeys(obj), []);
+
+obj[arrow_message_private_symbol] = 'bar';
+assert.strictEqual(obj[arrow_message_private_symbol], 'bar');
```

Comment:
I believe we want to check that `Reflect.ownKeys` returns nothing _after_ the assignment, before it's less interesting.

```suggestion

obj[arrow_message_private_symbol] = 'bar';
assert.strictEqual(obj[arrow_message_private_symbol], 'bar');
assert.deepStrictEqual(Reflect.ownKeys(obj), []);
```